common/test/data/scoreable/vertical/vertical1.xml

```diff
@@ -0,0 +1,3 @@
+<vertical display_name="CAPA">
```

Comment:
Nit: Consider renaming the vertical files to include the name of their embedded block.  For example:
vertical1.xml -> vertical_capa.xml 
vertical2.xml -> vertical_ora.xml (especially since it includes a whole bunch of ORA content)